src/utils/floating-ui.ts

```diff
@@ -380,11 +382,7 @@ export function connectFloatingUI(
 
   cleanupMap.set(
     component,
-    autoUpdate(referenceEl, floatingEl, () => {
-      if (component.open) {
```

Comment:
We should probably still skip repositioning if the component is not open.

Reply:
done!